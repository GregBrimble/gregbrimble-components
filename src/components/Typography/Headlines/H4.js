import React from 'react';

import styled from 'styled-components';
import {HeadlineSmall} from './shared';

const H4 = props => <HeadlineSmall as={styled.h4`
    font-size: 2.125rem;
    line-height: 2.5rem;
`} {...props} />;

export default H4;
