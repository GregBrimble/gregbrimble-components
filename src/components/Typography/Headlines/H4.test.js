import React from 'react';
import H4 from './H4';
import renderer from 'react-test-renderer';

describe('H4', () => {
    it('renders properly', () => {
        const tree = renderer
            .create(<H4>Headline 1</H4>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
