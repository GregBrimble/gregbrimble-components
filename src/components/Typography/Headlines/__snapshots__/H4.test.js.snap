// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H4 renders properly 1`] = `
<h4
  className="sc-bdVaJa sc-bwzfXH cSXMnb sc-htpNat lbBMXQ"
>
  Headline 1
</h4>
`;
